### PR TITLE
Add April 2022 monthly update

### PR DESCRIPTION
**Pull request by dholbach**
Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
Paging @staceypotter @kingdonb @pjbgf (and others of course!) to have look over this.

→ https://deploy-preview-911--fluxcd.netlify.app/blog/

File: content/en/blog/2022-05-04-april-update.md [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+#### Terraform-controller
+
+We have released TF-controller v0.9.4 which is a bug fixed release. We
+also added **cloud cost estimation** to our road map. Please feel free
+to give us feedback on how you would like this feature to be:
+
+- Issues here: <https://github.com/weaveworks/tf-controller/issues>, and
+- Discussions here: <https://github.com/weaveworks/tf-controller/discussions>
+
+#### Weave GitOps
```
Comment (sympatheticmoose) [inline content/en/blog/2022-05-04-april-update.md:157]: >Weave GitOps is a powerful, open source extension to Flux, which
provides insights into your deployments, and makes continuous delivery
with GitOps easier to adopt and scale across your teams. 
With the 0.7.0 release you can easily install alongside an existing Flux setup, adding
(or removing) Weave GitOps as a standard Helm resource. The new Web UI
provides deep insights into Flux resources to enable you to understand
their relationships and uncover their statuses.

Updating to reflect v0.8.0 release:  

Weave GitOps is a powerful, open source extension to Flux, which
provides insights into your deployments, and makes continuous delivery
with GitOps easier to adopt and scale across your teams. You can easily install alongside an existing Flux setup, adding (or removing) Weave GitOps as a standard Helm resource. 

The v0.8.0 release brings multi-namespace querying so you can see objects from across your cluster in the Web UI, several UI enhancements and bug fixes, as well as supporting the Source v1beta2 API - this breaking change means we now require Flux v0.29.0 or later.

Comment (dholbach) [reply content/en/blog/2022-05-04-april-update.md:157]: Super - added. Thanks @sympatheticmoose!

Comment (dholbach) [reply content/en/blog/2022-05-04-april-update.md:157]: https://deploy-preview-911--fluxcd.netlify.app/blog/2022/05/april-2022-update/#weave-gitops

**Review by pjbgf — APPROVED** [review]
LGTM!

File: content/en/blog/2022-05-04-april-update.md [hunk 2]
```diff
@@ -0,0 +1,611 @@
...
+
+We are looking forward to more collaboration together!
+
+#### Full GitOps Tutorial: Getting started with Flux CD
+
+{{< youtube 5u45lXmhgxA >}}
+
+This video is great for everyone who gets started, but also everyone who enjoys a story well-told.
+
+In this video, Anais Urlichs covers
```
Comment (kingdonb) [inline content/en/blog/2022-05-04-april-update.md:456]:
```suggestion
In this video, Anaïs Urlichs covers
```

File: content/en/blog/2022-05-04-april-update.md [hunk 3]
```diff
@@ -0,0 +1,611 @@
...
+
+- What is GitOps and how does Flux work
+- Flux installation
+- Managing Helm Charts with Flux
+- Managing Kubernetes Manifests with Flux
+- Setting up alerts with Flux
+
+Anais also sat down wrote this all up in [blog-post from](https://anaisurl.com/full-tutorial-getting-started-with-flux-cd/).
```
Comment (kingdonb) [inline content/en/blog/2022-05-04-april-update.md:464]:
```suggestion
Anaïs also sat down wrote this all up in [blog-post from](https://anaisurl.com/full-tutorial-getting-started-with-flux-cd/).
```

File: content/en/blog/2022-05-04-april-update.md [hunk 4]
```diff
@@ -0,0 +1,611 @@
...
+In the latest release, we have added checkboxes to enable Flux Subsystem
+in the Argo CD UI. We also have a [tutorial to use TF-controller with
+Flux Subsystem for
+Argo](https://flux-subsystem-argo.github.io/website/tutorials/terraform/).
+With this you have an alternative option to Crossplane to manage
+infrastructure.
+
+#### Terraform-controller
+
+We have released TF-controller v0.9.4 which is a bug fixed release. We
```
Comment (kingdonb) [inline content/en/blog/2022-05-04-april-update.md:150]:
```suggestion
We have released TF-controller v0.9.4 which is a bug fix release. We
```

**Review by kingdonb — APPROVED** [review]
LGTM 👍

File: content/en/blog/2022-05-04-april-update.md (hunk 4 above)
Comment (stefanprodan) [reply content/en/blog/2022-05-04-april-update.md:150]: Maybe "which is a patch release that comes with bug fixes"?

File: content/en/blog/2022-05-04-april-update.md (hunk 2 above)
Comment (dholbach) [reply content/en/blog/2022-05-04-april-update.md:422]: Yes, I wanted to do that, but that's not how it's spelled on the YouTube or website...

Comment (dholbach) [reply content/en/blog/2022-05-04-april-update.md:422]: Bildschirmfoto von 2022-05-03 16-01-38

File: content/en/blog/2022-05-04-april-update.md (hunk 4 above)
Comment (dholbach) [reply content/en/blog/2022-05-04-april-update.md]: :heavy_check_mark:

**Pull request merged**